packages/at_lookup/lib/src/at_lookup.dart

```diff
@@ -23,4 +24,7 @@ abstract class AtLookUp {
   Future<List<String>> scan({String? regex, String? sharedBy});
 
   Future<String?> executeVerb(VerbBuilder builder, {bool sync = false});
+
+  /// Set an instance of [AtChops] for signing operations during authentication
+  void setAtChops(AtChops atChops);
```

Comment:
Actually, just one thing - can we make this a standard Dart setter?

Reply:
done